mxpy-up.py

```diff
@@ -21,9 +21,15 @@ def main():
     parser.add_argument("--from-branch", help="use a branch of multiversx/mx-sdk-py-cli")
     parser.add_argument("--not-interactive", action="store_true", default=False)
     parser.add_argument("--verbose", action="store_true", default=False)
+    parser.add_argument("--ignore-deprecation", action="store_false", default=True, help="'mxpy-up.py' is obsolete, install using 'pipx': https://docs.multiversx.com/sdk-and-tools/sdk-py/installing-mxpy/#install-using-pipx")
     parser.set_defaults(modify_path=True)
     args = parser.parse_args()
 
+    if args.ignore_deprecation:
```

Comment:
Ignore deprecation should allow one to install.

The flag should be required. That is, when the user provides it, can install (but sees warnings). If not provided, the user should not be able to install.

Does it make sense?

Reply:
It does make sense, but that's what is does. It's not required in the `parser.add_argument(...)` so we can display the warnings, but the code does not execute if the flag is not provided. It's `False` by default and if it's not provided, the `if` statement executes and the `return` statement stops the installation. If provided, the installation process works and the warning is displayed at the end of execution. Let me know if I misunderstood.

Reply:
Oh, now I see.

Can we somehow convert the `return` on line 31 from a happy return to a failure? So that CI workflows that still use mxpy-up don't see a "success" (since it's a failure due to not providing the flag)?